test/vstest.console.UnitTests/Internal/ConsoleLoggerTests.cs

```diff
@@ -334,7 +394,7 @@ public void TestResultHandlerShouldWriteToConsoleShouldShowPassedTestsForNormalV
         }
 
         [TestMethod]
-        public void TestResultHandlerShouldShowStdOutMsgOfPassedTestIfVerbosityIsNormal()
+        public void TestResultHandlerShouldShowNotStdOutMsgOfPassedTestIfVerbosityIsNormal()
```

Comment:
>TestResultHandlerShouldShowNotStdOutMsgOfPassedTestIfVerbosityIsNormal [](start = 20, length = 70)

Nit: Should*Not*Show
please change the name for test following this one as well.